### ISSUE TITLE
Need `InteractiveUtils` in Project.toml

### ISSUE THREAD

**Issue by Datseris** [fbanning/OSMMakie.jl#7]
Just a typicallity:
```
┌ Warning: Package OSMMakie does not have InteractiveUtils in its dependencies:
│ - If you have OSMMakie checked out for development and have
│   added InteractiveUtils as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with OSMMakie
└ Loading InteractiveUtils into OSMMakie from project dependency, future warnings for OSMMakie are suppressed.
WARNING: Method definition registration_setup!(Any, GraphMakie.GraphInteraction) in module GraphMakie at C:\Users\datse\.julia\packages\GraphMakie\tEQ8U\src\interaction.jl:45 overwritten in module OSMMakie at C:\Users\datse\.julia\dev\OSMMakie\src\interaction.jl:6.
  ** incremental compilation may be fatally broken for this module **
```
Standard library packages still need to be added to project.toml

**Comment by fbanning** [comment]
Oops, didn't know that. Will fix it on `interaction` branch because that's where I've added the dep.

**Comment by fbanning** [comment]
Fixed in d854f7262e42f7670035c8b99a1eade3acfaf010
